docs/lang/articles/reference.md

```diff
@@ -301,6 +301,17 @@ is evaluated to the value of that expression.
 
 ### Assignment statements
 
+```
```

Comment:
"assignment expression" should be added as an individual subsection under "expressions"